pages/jobsearchingandapplications/CraftingaTechfocusedResume.mdx

```diff
@@ -28,11 +33,39 @@ d) Highlighting Transferable Skills
 e) Education and Certifications
 
 - Include your educational background, focusing on degrees, courses, or certifications relevant to your desired tech role.
-- If applicable, list any tech-related bootcamps, workshops, or online courses you have completed to further demonstrate your commitment to skill development in the tech industry.
+- Do not List Out Any Tech Bootcamp courses or online courses in your resume, this gives the impression that you lack real world collaboration experience to function in a workplace.
```

Comment:
What will be best for a beginner without real-world experience to add? I don't think having a list of courses completed and certifications will be a bad idea. As a newbie, they probably don't have real-world experience yet. I will suggest we should keep the first point.